x/stake/querier/queryable.go

```diff
@@ -119,6 +123,23 @@ func queryValidator(ctx sdk.Context, cdc *codec.Codec, req abci.RequestQuery, k
 	return res, nil
 }
 
+func queryValidatorDelegations(ctx sdk.Context, cdc *codec.Codec, req abci.RequestQuery, k keep.Keeper) (res []byte, err sdk.Error) {
+	var params QueryValidatorParams
+
+	errRes := cdc.UnmarshalJSON(req.Data, &params)
```

Comment:
I'd consider using the var name `err` here and removing names from the function return arguments - just a thought

Reply:
#2788